src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
 export interface Figure {
-
+  color: 'red' | 'green' | 'blue';
+  getArea(): number | Error
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public aSide: number,
+    public bSide: number,
+    public cSide: number,
```

Comment:
You can simply name it:
```suggestion
    public a: number,
    public b: number,
    public c: number,
```

And then use it directly in class body